experiments/list_unsupported_report.sh

```diff
@@ -134,7 +134,9 @@ sed -n 's/^.*:[0-9]*:[0-9]*: error: //p' "$raw_input_file" | \
          awk '/^ *[0-9]+ .*$/ { \
             count=$1; \
             raw=$0; sub(/^ *[0-9]+ /, "", raw); \
-            redacted=raw; gsub(/"[^"]+"/, "\"REDACTED\"", redacted); \
+            redacted=raw; \
+            gsub(/"[^"]+"/, "\"REDACTED\"", redacted); \
+            gsub(/at ([a-zA-Z0-9\.\-_]+:[0-9]+)/, "at REDACTED", redacted); \
```

Comment:
Is this not affecting the golden results? If it is, why is there no change to the golden results _after_ this?

Reply:
I think we don't actually have any of the particular messages that were previously missed appearing in any of the CI projects. This was tested on the example output given by the original bug reporter.